pkg/operator/ceph/csi/secrets.go

```diff
@@ -148,20 +153,20 @@ func createOrUpdateCSISecret(clusterInfo *client.ClusterInfo, csiRBDProvisionerS
 		s := &v1.Secret{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      secretName,
-				Namespace: clusterInfo.Namespace,
+				Namespace: os.Getenv(k8sutil.PodNamespaceEnvVar),
```

Comment:
This should not change, the secret should be created in the ceph cluster namespace not in operator namespace